.github/workflows/pull_request.yml

```diff
@@ -8,39 +8,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Push Docker Image to Github Registry
-        uses: whoan/docker-build-with-cache-action@v5
-        with:
-          username: "${{ github.actor }}"
-          password: "${{ secrets.GITHUB_TOKEN }}"
-          image_name: ${{ secrets.flytegithub_repo }}/flytepropeller
-          image_tag: ${{ github.sha }}
-          push_git_tag: true
-          registry: docker.pkg.github.com
+        name: "Checkout"
+
       - uses: engineerd/setup-kind@v0.4.0
       - name: End2End
-        env:
-          DOCKER_USERNAME: ${{ github.actor }}
-          DOCKER_PASSWORD: "${{ secrets.GITHUB_TOKEN }}"
         run: |
+          docker build -t lyft/flytepropeller:${{ github.sha }} .
```

Comment:
Can we continue to use the docker with cache action? we don't need to rebuild the image from scratch every time... specially pulling dependencies.

If my guess is correct, we should still be able to directly `kind load docker-image docker.pkg.github.com/${{PROPELLER}}` because it'll exist in the machine cache... I could be wildly wrong though :)

Reply:
Is this still needed?